pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "airtop"
-version = "0.0.23"
+version = "0.0.24"
```

Comment:
We need to manually increase the version number as this file is currently ignored by fern.